conftest.py

```diff
@@ -16,6 +16,14 @@
 import pytest_pyodide.runner
 from pytest_pyodide.utils import package_is_built as _package_is_built
 
+pytest_pyodide.runner.CHROME_FLAGS.extend(
+    [
+        "--enable-features=WebAssemblyExperimentalJSPI",
+        "--enable-experimental-webassembly-features",
+    ]
+)
+pytest_pyodide.runner.NODE_FLAGS.extend(["--experimental-wasm-stack-switching"])
```

Comment:
So the idea is that we are still running Firefox tests without JSPI right? 

I wonder if it wouldn't be good to still run some tests for chrome (and possibly node) without this flag. Since that's the default configuration where most users are currently.

Reply:
I think it would be good to run the fpcast tests, the C++ exception handling tests, the ctypes tests, the rust tests, and the recursion limit tests in the core test suite in both configurations. The remaining tests can be run in just one or the other.

Reply:
I'll fix this in a followup PR.